packages/output/src/node/output-channel-service-impl.ts

```diff
@@ -0,0 +1,112 @@
+/********************************************************************************
```

Comment:
This PR does not just send log messages to the output view, it adds support so that text can be written to output channels in the backend.  This was deliberately put in node so it is generally available.  For example if @NitinKhandelwal1947 wants to write the output from the script directly to an output channel then this can be done even in the browser version.

Reply:
> For example if @NitinKhandelwal1947 wants to write the output from the script directly to an output channel

That would be visible for all connected frontends, right? We decided to have the logging for the electron version.

Reply:
No, the logging is configured only for Electron.  The support for writing to output channels at the backend is available to both backend browser and backend electron.  So if an extender wants to write logs from the backend (or any other data from the backend) to an output channel in the browser version then they can easily do so, but it would not be the default behavior.